sdk/networkanalytics/azure-resourcemanager-networkanalytics/src/test/java/com/azure/resourcemanager/networkanalytics/NetworkAnalyticsManagerTests.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager.networkanalytics;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.core.test.TestBase;
+import com.azure.core.test.annotation.DoNotRecord;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import org.junit.jupiter.api.Test;
+
+public class NetworkAnalyticsManagerTests extends TestBase {
+    private NetworkAnalyticsManager networkAnalyticsManager = null;
+
+    @Override
+    public void beforeTest() {
+        final TokenCredential credential = new DefaultAzureCredentialBuilder().build();
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        networkAnalyticsManager = NetworkAnalyticsManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+    }
+
+    @Test
+    @DoNotRecord(skipInPlayback = true)
```

Comment:
use `@LiveOnly` -- this is newly added in azure-core-test

Reply:
Fixed in the new version.